executor/aggregate_test.go

```diff
@@ -752,3 +752,17 @@ func (s *testSuite1) TestIssue10608(c *C) {
 	tk.MustQuery("select (select group_concat(concat(123,'-')) from t where t.a = s.b group by t.a) as t from s;").Check(testkit.Rows("123-", "123-"))
 
 }
+
+func (s *testSuiteAgg) TestPR15242ShallowCopy(c *C) {
```

Comment:
Please fix this issue:"executor/aggregate_test.go:756:10: undefined: testSuiteAgg"

Reply:
ok, tks.